#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enabled", bool_t, 0, "Whether to apply this plugin or not", True)
gen.add("one_zone", bool_t, 0, "Save only one zone of navigation or not", True)
gen.add("clear_obstacles", bool_t, 0, "Whether to clear obstacles or not", True)
gen.add("map_frame", str_t, 0, "Name of the map frame", "odom")
gen.add("base_frame", str_t, 0, "Name of the base frame of the robot", "base_link")

exit(gen.generate("vision_costmap_layer", "vision_costmap_layer", "VisionLayer"))